src/index.ts

```diff
@@ -150,7 +169,7 @@ export default class IncludeFragmentElement extends HTMLElement {
     }
   }
 
-  #getData(): Promise<string> {
+  #getData(): Promise<string | CSPTrustedHTMLToStringable> {
```

Comment:
The return type is inaccurate. I've created https://github.com/github/include-fragment-element/pull/84 to tackle that.

Reply:
I've merged #84 into this, trying to keep the types as consistent as possible.